### PR TITLE
Cleanup: Lua bindings

### PR DESCRIPTION
**Pull request by slipher**

### CODE REVIEW

**Review by DolceTriade — APPROVED** [review]

File: src/cgame/rocket/rocket.cpp [hunk 1]
```diff
@@ -482,7 +479,7 @@ void Rocket_Update()
 	{
 		hudContext->Update();
 	}
-	Rml::Lua::Timer::Update(rocketInfo.realtime);
+	Timer::Update(rocketInfo.realtime);
```
[the remark this answers is not in the document]
Comment (slipher) [reply src/cgame/rocket/rocket.cpp:482]: Can we nuke it? Nothing is using the Timer API.

[the remark this answers is not in the document]
Comment (DolceTriade) [reply src/cgame/rocket/rocket.cpp:482]: Id prefer not to. I'll definitely use it for scripting with sgame lua

[the remark this answers is not in the document]
Comment (slipher) [reply src/cgame/rocket/rocket.cpp:482]: I moved the class definition to the .cpp and defined a namespaced function to update the timers.

[the remark this answers is not in the document]
Comment (slipher) [reply src/cgame/rocket/rocket.cpp]: > I moved the class definition to the .cpp and defined a namespaced function to update the timers.

Oops, I never actually pushed the code. Now it's there.

**Pull request merged**